### PR TITLE
Adding Missing Portals

### PR DESCRIPTION
**Pull request by JohnnyPlayy**
* **Addressed Issue(s)**: 

* **Server Mode**: Both

* **Description of Pull Request**: 
Information taken directly from KRO
image
I noticed that there is a missing NPC > [Missing script](https://www.pastiebin.com/5d9688bec5ddb)

### CODE REVIEW

**Comment by Questune09** [comment]
`yuno_in04,186,21,1	script	Stylist#yuno	4_M_BARBER,{
	mes "[Maxi]";
	mes "Nice to meet you.";
	mes "Schwarzwald's best stylist Would you like to let your maxi transform yourself?";
	switch(select("Change the style:Finish conversation")) {
	case 1:
		OpenStyleUI();
		close;
	}
	case 2:
		mes "[Maxi]";
		mes "Come again anytime.";
		close;
	}
}`

just need to double check the script not sure with this.

**Comment by JohnnyPlayy** [comment]
@Lemongrass3110 
Yuno's portal i added
```
yuno,195,87,0	warp	yun16	1,1,yuno_in04,186,28
yuno_in04,192,28,0	warp	yun17	1,1,yuno,198,87
```

**Comment by JohnnyPlayy** [comment]
Done

**Comment by aleos89** [comment]
Are those two last wraps gone now (the old ones)? If so they should be added to the pre-re file.

File: npc/warps/cities/yuno.txt [hunk 1]
```diff
@@ -87,8 +87,8 @@ yuno_in01,88,36,0	warp	yun59	1,1,yuno_in01,173,34
 yuno_in01,176,34,0	warp	yun60	1,1,yuno_in01,91,36
 yuno_in02,82,14,0	warp	yun61	1,1,yuno_in05,192,194
 yuno_in05,196,194,0	warp	yun62	1,1,yuno_in02,85,14
-yuno,245,147,0	warp	yun65	1,1,yuno_in01,101,88
-yuno_in01,101,85,0	warp	yun66	1,1,yuno,245,144
+yuno,195,87,0	warp	yun63	1,1,yuno_in04,186,28
```
[the remark this answers is not in the document]
Comment (JohnnyPlayy) [reply npc/warps/cities/yuno.txt]: Portals yun65 and yun66 were removed by my error

[the remark this answers is not in the document]
Comment (JohnnyPlayy) [reply npc/warps/cities/yuno.txt]: I checked this portal on Iro and it was apparently there

[the remark this answers is not in the document]
Comment (Lemongrass3110) [reply npc/warps/cities/yuno.txt]: iRO is also renewal. As i said we need to check in AEGIS for pre-renewal. :-)

[the remark this answers is not in the document]
Comment (Lemongrass3110) [reply npc/warps/cities/yuno.txt]: > 
> 
> Portals yun65 and yun66 were removed by my error

Thanks for adding them back!

[the remark this answers is not in the document]
Comment (Lemongrass3110) [reply npc/warps/cities/yuno.txt]: Ah now that I see it. yun63 is already taken, see line 96 below!
Could you change them to yun78 and yun79?

**Pull request merged**